code/API_definitions/device-status.yaml

```diff
@@ -496,44 +564,63 @@ components:
         notificationAuthToken:
           type: string
           example: "c8974e592c2fa383d4a3960714"
-          description: | 
+          description: |
             OAuth2 token to be used by the callback API endpoint. It MUST be indicated within HTTP Authorization header e.g. Authorization: Bearer $notificationAuthToken
 
-    EventSubscriptionDetail:
+    SubscriptionDetail:
       description: The detail of the requested event subscription
       type: object
       required:
-        - eventType
+        - type
       properties:
         device:
-          $ref: '#/components/schemas/Device'
-        eventType:
-          $ref: '#/components/schemas/RoamingEventType'
+          $ref: "#/components/schemas/Device"
+        type:
+          $ref: "#/components/schemas/RoamingEventType"
 
     RoamingEventType:
       type: string
       description: |
-        ROAMING_STATUS - Event triggered when the device switch from roaming ON to roaming OFF and conversely
+        roaming-status - Event triggered when the device switch from roaming ON to roaming OFF and conversely
```

Comment:
I think it would be better to use SubscriptionType or SubscriptionEventType instead of RoamingEventType. 
- to better distinguish between subscription event type and notification event type
- as a preparation for other type of events